modules/prebidServerBidAdapter.js

```diff
@@ -19,15 +19,32 @@ const DEFAULT_S2S_CURRENCY = 'USD';
 const DEFAULT_S2S_NETREVENUE = true;
 
 let _s2sConfig;
+
+const s2sDefaultConfig = {
+  enabled: false,
+  timeout: 1000,
+  maxBids: 1,
+  adapter: 'prebidServer'
+};
+
 config.setDefaults({
-  's2sConfig': {
-    enabled: false,
-    timeout: 1000,
-    maxBids: 1,
-    adapter: 'prebidServer'
-  }
+  's2sConfig': s2sDefaultConfig
 });
 
+// accountId and bidders params are not included here, should be configured by end-user
+const avail_vendor_defaults = {
+  'appnexus': {
+    adapter: 'prebidServer',
+    cookieSet: true,
+    cookieSetUrl: '//acdn.adnxs.com/cookieset/cs.js',
+    enabled: true,
+    endpoint: '//prebid.adnxs.com/pbs/v1/auction',
+    syncEndpoint: '//prebid.adnxs.com/pbs/v1/cookie_sync',
+    timeout: 1000
+  },
+  'rubicon': {}
```

Comment:
Rubicon's PBS defaults:
```
adapter: 'prebidServer',
cookieSet: true,
cookieSetUrl: 'https://secure-assets.rubiconproject.com/utils/cookieset/cs.js',
enabled: true,
endpoint: 'https://prebid-server.rubiconproject.com/auction',
syncEndpoint: 'https://prebid-server.rubiconproject.com/cookie_sync',
timeout: 500
{code}

Reply:
Hi @bretg, Thanks for submitting the information.  Just to confirm, you do want to have hard-coded secure protocol for the URLs?  For the AppNexus vendor, we were leaving them out to be dynamically populated based on the page.  Thanks.

Reply:
Hey @bretg, I've submitted the Rubicon config as is for now (as I had some other changes I wanted to submit).  If you want to have the values changed in lieu of the above (or any other reason), please feel free to let me know and I can make the updates.